### PR TITLE
Fixed paths for label tests.

### PR DESCRIPTION
**Pull request by nsndeck**

### CODE REVIEW

File: apps/tests/ui/label/label-tests.ts [hunk 1]
```diff
@@ -477,7 +478,7 @@ export var testLocalTextAlignmentFromCssWhenAddingCssFileAllSelectorsAreApplied
         view.id = "testLabel";
         var page = <page.Page>views[1];
         page.addCss("#testLabel { text-align: " + expectedTextAlignment + "; }");
-        page.addCssFile("/tests/ui/label/label-tests.css");
+        page.addCssFile(fs.path.join(__dirname,"/label-tests.css"));
```
Comment (ErjanGavalji) [inline apps/tests/ui/label/label-tests.ts]: a space after the comma here? ;-)

**Comment by ErjanGavalji** [comment]
:+1:

**Comment by dtopuzov** [comment]
:100:

**Comment by enchev** [comment]
:+1:

**Pull request merged**